### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by wolterbwambale**
I have built my capstone project  has the following:

   home page, about page. with media queries of less than 768px to a wider screen.
it has the interaction of links i.e home page and about page on both mobile and desktop versions.
The mobile menu has a close (X) button that closes the menu and   Featured speakers are created dynamically in JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,52 +114,34 @@ Example:
...
-> Write a message to encourage readers to support your project
-
-If you like this project...
+Give a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (mwanawabangona) [inline README.md:136]: - [ ] `required`
- [ ] On the  "Acknowledgements" section this where  you give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🙏 
- [ ] Please make the changes 🥂

File: index.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+    <div class="programs">
+        <h2 class="main-pro">Main Programes</h2>
+        <hr class="line-h"/>
+    <div class="prog">
+        <div class="sub-cont">
+        <div class="imgcont">
+            <img src="./img/main-icon/Screenshot (207).png" alt="lecture">
+        </div>
+        <div class="text">
+            <span>lecture</span>
```
Comment (mwanawabangona) [inline index.html:110]: - [ ] `required`
- [ ] Please make the border-boxes same size and and aligned like the origin template.
- [ ] please see screenshot below

`your template`
image

`orignal template`
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+        </header>
+        
+        <header class="header-desk">
+            
+            <nav class="heads-all">
+                <ul>
+                    <li class="logo-img">
+                        <a href="./index.html">
+                            <img src="./img/CC_Global_Summit_2015 logo.png" alt="collision logo">
+                        </a>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Please make the first navbar black like the template
- [ ] also the logo should be in line with where you start the paragraph `face to face with gorillas`
- [ ] and I suggest you increase the fonts like the original template
- [ ] please screenshots below

`your template`
image

`original template`
image

File: README.md [hunk 4]
```diff
@@ -1,233 +1,110 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+[Live Demo]()
 
-- [Live Demo Link](https://google.com)
```
Comment (mwanawabangona) [inline README.md:114]: - [ ] `required`
- [ ] Please deployed your project as required.

`see the error i'm getting`
image

File: README.md [hunk 5]
```diff
@@ -1,233 +1,110 @@
...
+
+<!--Video-->
+## Video Presentation <a name="video-presentation"></a>
+[Click Here for Video Presentation](https://www.loom.com/share/37fdc5ee0b2b4ab9994e54aa0f504f6a)
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] I'm a bit confused here, your profile shows you are male but the video presentation is done by a female.
- [ ] Please help me understand

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @wolterbwambale,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @wolterbwambale , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**